Kerberos.NET/Entities/Pac/ClaimEntry.cs

```diff
@@ -38,7 +38,7 @@ public void Marshal(NdrBuffer buffer)
             buffer.WriteDeferredStructUnion(this);
         }
 
-        public string Id { get; private set; }
+        public string Id { get; set; }
```

Comment:
This was set as private originally because I wasn't convinced the overall Claim structure would roundtrip correctly through the NDR encoder. Based on the bug you're saying you set this through reflection. Are you using this against Windows and showing it working?

Reply:
We have not tested the whole story (and we most likely will not, because our use case is somewhat limited) but we can validate that the constructed tickets are successfully serialized and deserialized on Windows when imported with the `LsaInterop`.

Reply:
Good enough for me.